EnglishFilter/sections/antiadblock.txt

```diff
@@ -545,6 +545,8 @@ kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74902
+ robot-forum.com#%#//scriptlet("abort-current-inline-script", "$", "offsetHeight")
```

Comment:
Re-write the rule a bit `robot-forum.com#%#//scriptlet("abort-current-inline-script", "$", "adblock")`

Reply:
Why though?

Reply:
We should use simple and clear rules, if possible.

Reply:
Okay.